src/models/sqlOutputContentProvider.ts

```diff
@@ -33,6 +33,7 @@ export class QueryRunnerState {
 class ResultsConfig implements Interfaces.IResultsConfig {
     shortcuts: { [key: string]: string };
     messagesDefaultOpen: boolean;
+    resultsFontSize: number;
```

Comment:
You shouldn't need to define these here - they're already a part of the interface definition

Reply:
I added to the interface in the same PR

Reply:
I'm saying that because it's defined in the interface you don't need to redefine it here - it'll inherit them from the interface already. You would only need to do something if you're adding actual implementation such as a method definition.

Reply:
This is implementing the interface, not extending it, so we do need to specify the field

Reply:
Argh, yeah you're completely right. Dunno what I was thinking - sorry!